src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/internal/impl/CaseWorkerInternalApiClientImpl.java

```diff
@@ -0,0 +1,66 @@
+package uk.gov.hmcts.reform.cwrdapi.controllers.internal.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.context.request.RequestContextHolder;
+import org.springframework.web.context.request.ServletRequestAttributes;
+import org.springframework.web.util.UriComponents;
+import org.springframework.web.util.UriComponentsBuilder;
+import uk.gov.hmcts.reform.cwrdapi.controllers.internal.CaseWorkerInternalApiClient;
+
+import java.util.List;
+import javax.servlet.http.HttpServletRequest;
+
+import static java.util.Objects.nonNull;
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+
+@Service
+public class CaseWorkerInternalApiClientImpl implements CaseWorkerInternalApiClient {
+
+    public static final String SERVICE_AUTHORIZATION = "ServiceAuthorization";
+    public static final String AUTHORIZATION = "Authorization";
```

Comment:
move these constants in util or constant class may be we can re use in other places if required.

Reply:
sure